### PR TITLE
Remove the container initializers per platform.

### PR DESCRIPTION
**Pull request by calavera**
By removing deprecated volume structures, now that windows mount volumes we don't need a initializer per platform.

Signed-off-by: David Calavera david.calavera@gmail.com

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM
ping @jhowardmsft

File: daemon/container.go [hunk 1]
```diff
@@ -77,6 +77,20 @@ type CommonContainer struct {
...
+// newBaseContainer creates a new container with its
+// basic configuration.
+func newBaseContainer(id, root string) *Container {
+	return &Container{
+		CommonContainer: CommonContainer{
+			ID:           id,
+			State:        NewState(),
+			execCommands: newExecStore(),
+			root:         root,
+			MountPoints:  make(map[string]*volume.MountPoint),
```
Comment (jhowardmsft) [inline daemon/container.go]: Looks good except possibly for this line - it wasn't initialized previously on Windows.

Comment (LK4D4) [inline daemon/container.go]: I think it's ok, it'll be `{}` instead of `null` in json. Am I right, @calavera ?

**Pull request merged**